app/main/views/marketplace.py

```diff
@@ -118,7 +116,11 @@ def list_opportunities(framework_framework):
 
     api_result = form.get_briefs()
 
-    briefs = api_result["briefs"]
+    briefs = [{
```

Comment:
The content does not contain everything we need because we need attributes such as `publishedAt` and `title`. `publishedAt` is not a question in any manifest as it is an attribute on the brief only. `title` is a question for the edit_brief manifest but not for the display_brief manifest.

We have copied the pattern for a single brief of doing this - https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/app/main/views/marketplace.py#L92

However we did spend a bunch of time deliberating what the best option was as this does feel a bit complicated. We did consider the simplest of overriding `brief['specialistRole'] = thing_formatted_by_content_loader` before passing the brief into the template but thought this seemed to be more presentation related and should maybe live in the template which therefore requires passing both the brief and the content in.

Any suggestions or content loader tricks we aren't aware of?